### PR TITLE
i18n support and general code refactoring

### PR DESCRIPTION
**Pull request by Gramatiik**
This PR covers two parts : 
- **i18n support** (with fr_FR and en_US translations).
- Code refactoring to **follow the classic resource structure** (config file, locales folder, etc.).

If you have any questions/suggestions, don't hesitate to comment this PR 😄

### CODE REVIEW

**Comment by ElPumpo** [comment]
Remove all the print debugging lines as they create their own spam. The language files are not following the standard we are using. Keep it two letters (country) like any other esx script. I know that the ISO culture name is better but that's just how every other script is developed and we cannot have differences.

**Comment by Gramatiik** [comment]
print debugs were already in the original script, and quite useful for server side debugging.
Maybe having a debug option would be nice.

For locales, i'll change it to the `fr` and `en`, if this sticks more to _every other scripts_.

File: config.lua [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Config = {}
 
 Config.Locale = 'en'
+Config.Debug = false
```
Comment (ElPumpo) [inline config.lua]: Use [ESX.Trace(msg)](https://esx-org.github.io/es_extended/server-functions/esx.trace/) instead

Comment (Gramatiik) [reply config.lua]: Didn't knew about this, I'll switch to that to avoid this useless debug flag.

**Comment by ElPumpo** [comment]
Sorry for being so picky, but thanks for your contributations!

**Comment by Gramatiik** [comment]
No problem @ElPumpo ! Being demanding makes a better code ;)

**Pull request merged**